### PR TITLE
FA3.5.0 Revive

### PR DESCRIPTION
**Pull request by SamLex**
Current "issues" with Revive:
* Everyone can fully heal everyone
* Little feedback when downed in a vehicle
* No animation while healing

Things this pull request changes:
* Restructure of medical code; its a bad habit of mine
* "Pull out wounded" is now a hold-action; consistence with revive action
* Virtual camera no longer used in vehicle, player downed as if not in a vehicle; serves same purpose (can't control vehicle, get out, etc.), less hacky, **allows players to bleed out in vehicles**
* Restored vanilla behaviour of FAKs and medkits in normal use
* Reviving a player only heals them as much as a FAK instead of full; full heal for medics would be nice but I believe it to be impossible without basically rewriting BIS Revive.
* As well as removing "Force Respawn" hold-action while downed, a "Wait to be revived" "infinite" hold-action has been added; allows use of animated central icon (blinks skull when close to death, blinks medic symbol when being revived)
* Fixes a bug that allows multiple "Pull out" actions on the same vehicle

How this relates to "issues":
* Basically fixed: at worst everyone can heal everyone to FAK level and medics can heal as normal for non-downed players.
* Now get grey-out screen effect, and "death" animation in vehicles with interiors.
* While not an animation, the animated icons on both ends gives a reviving and downed player some feedback as to what is happening.

### CODE REVIEW

File: f/medical/fn_addPullOutAction.sqf [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		}
+	} forEach crew _target;
+};
+
+_actionId = [
+	_vehicle,
+	"Pull out wounded",
+	"\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_unbind_ca.paa",
+	"\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_unbind_ca.paa",
+	"(_this distance _target < 5) && {[_target] call f_fnc_vehicleHasWounded}",
```
Comment (SamLex) [inline f/medical/fn_addPullOutAction.sqf]: Note to later self: this needs a not in vehicle check. Currently getting into the vehicle with a downed player shows the actions.

**Comment by SamLex** [comment]
This was successfully tested in session last night with no issues related to these changes. The only issue encountered was that the wounded post-process effect sometimes persists into spectator, but this seems to be a known issue that occurs in the official End Game as well.

So I think this and then https://github.com/Raptoer/F3/pull/86 can be merged, and FA3.5.0 released.
